src/main/java/org/hl7/davinci/ehrserver/authproxy/AuthProxy.java

```diff
@@ -143,7 +143,7 @@ public void authSync(@PathVariable String launch, @RequestParam Map<String, Stri
    */
   private String _parseRedirect(Map<String, String> reqParamValue, HttpServletRequest request) {
     String currentRedirectURI = reqParamValue.get("redirect_uri");
-    String finalRedirectURI = "http://" +request.getLocalName() + ":" + request.getLocalPort() + "/test-ehr/_auth/" + reqParamValue.get("launch") + "?redirect_uri=" + currentRedirectURI;
+    String finalRedirectURI = "http://" + "localhost" + ":" + request.getLocalPort() + "/test-ehr/_auth/" + reqParamValue.get("launch") + "?redirect_uri=" + currentRedirectURI;
```

Comment:
need to environmentalize this variable before merging, request.getLocalName() returns the containers network host name id instead of localhost. Hard coded string change is temporary solution